js/main.js

```diff
@@ -17,6 +38,40 @@ const getRandomFloatInt = (min, max, precision) => {
   throw new Error('getRandomFloatInt - диапазон или точность не соответствуют условию');
 };
 
-getRandomRoundedInt(1, 3);
-getRandomFloatInt(1.2, 1.3, 2);
+const getRandomArrayElement = (array) => array[getRandomRoundedInt(0, array.length - 1)];
+
+const getRandomArrayElements = (array) => array.slice(getRandomRoundedInt(1, array.length - 1));
+
+const createRandomOffer = () => {
+  let locationX = getRandomFloatInt(35.65000, 35.70000, 5);
+  let locationY = getRandomFloatInt(139.70000, 139.80000, 5);
+  return {
+    author: {
+      avatar: `img/avatars/user0${getRandomRoundedInt(1, TOTAL_USERS)}.png`,
+    },
+    location: {
+      x: locationX,
+      y: locationY,
+    },
+    offer: {
+      title: 'Заголовок',
+      address: `${locationX},${locationY}`,
+      OFFER_PRICE: getRandomRoundedInt(OFFER_PRICE.min, OFFER_PRICE.max),
+      type: getRandomArrayElement(APARTMENT_TYPE),
+      rooms: getRandomRoundedInt(ROOMS_NUMBER.min, ROOMS_NUMBER.max),
+      guests: getRandomRoundedInt(GUESTS_NUMBER.min, GUESTS_NUMBER.max),
+      checkin: getRandomArrayElement(CHECKIN_TIME),
+      checkout: getRandomArrayElement(CHECKOUT_TIME),
+      features: getRandomArrayElements(APARTMENT_FEATURES),
+      description: 'Описание',
+      photos: getRandomArrayElements(APARTMENT_PHOTOS),
+    },
+  };
+};
+
+const getRandomOffers = () => {
+  let randomOffers = new Array(10).fill(null).map(createRandomOffer);
+  return randomOffers;
+};
```

Comment:
Можно так
```suggestion
const getRandomOffers = () => new Array(10).fill(null).map(createRandomOffer);
```

Создание переменных должно быть оправданно, т.к. новая переменная занимает место в памяти, пусть даже временно. Вдобавок разработчик читающий этот код не найдет причину зачем тут создается новая переменная. 
Это пример очень простой, но в реальных больших проектах это может усложнять понимание кода